src/base/operations/initiate_kyc_recovery_builder.js

```diff
@@ -0,0 +1,44 @@
+import { default as xdr } from '../generated/xdr_generated'
+import { BaseOperation } from './base_operation'
+import { Keypair } from '../keypair'
+
+export class InitiateKYCRecoveryBuilder {
+  /**
+     * Creates operation to create KYC request
+     * @param {object} opts
+     * @param {string} opts.targetAccount - target account to initiate recovery for
+     * @param {string} opts.signer
```

Comment:
please describe the role of this `opts.signer` parameter. the signer is to be added? or the signer is assumed to exist?